readme.md

```diff
@@ -35,13 +35,15 @@ Renovate was released in 2017 and already is widely used in the developer commun
 
 ## Using Renovate
 
-The easiest way to use Renovate if you are hosted on GitHub.com or GitLab.com is to install the hosted Renovate app.
+The easiest way to use Renovate if you are hosted on GitHub.com or GitLab.com or DevOps (microsoft) is to install the hosted Renovate app.
 
 For GitHub, go to [https://github.com/apps/renovate](https://github.com/apps/renovate) to install
 it now. A GitHub Marketplace plan is required only for private repositories. [More details on the GitHub App installation](https://docs.renovatebot.com/install-github-app/).
 
 For GitLab, log into [app.renovatebot.com/dashboard](https://app.renovatebot.com/dashboard) for a convenient way to add Renovate Bot to your project. Currently the GitLab app is free for both public and private repositories. [More details on the GitLab App installation](https://docs.renovatebot.com/install-gitlab-app/).
 
+For DevOps (microsoft), go to the marketplace and add the [RenovateMe](https://marketplace.visualstudio.com/items?itemName=jyc.vsts-extensions-renovate-me) addon to your organization. Then create a pipeline with this new task and renovate will do his job.
```

Comment:
I would write it this like:

> For Azure DevOps, visit the Visual Studio Marketplace and install the [Renovate Me](https://marketplace.visualstudio.com/items?itemName=jyc.vsts-extensions-renovate-me) extension in your organisation. From there you can create a pipeline with the `RenovateMe` task.

From the source of your extension, it looks like the build task is just called `RenovateMe`, is that right? Also, as Azure DevOps support is a little more complicated, we might want to create additional help pages.

Reply:
As you saw, there is already a ```readme.md``` there to explain what to do with this Task in the pipeline. So maybe we can put here: 

>For Azure DevOps, visit the Visual Studio Marketplace and install the Renovate Me extension in your organisation. From there you can create a pipeline with the RenovateMe task as described in the addon description.

Reply:
Yeah, sounds good.

One thing though, you might want to provide a YAML build sample. I think most new builds are created this way, as opposed to using the older GUI builds.